### ISSUE TITLE
A question about deepwalk

### ISSUE THREAD

**Issue by JiayinZheng** [Abigale001/TMER#6]
In data_process.py, you use all the user-item relations (both the training set and the test set) to build graphs and embed nodes(in embed_nodes.py). However, I'm now questioning this process since the user-item relations of the test set are also included.  Looking forward to your reply and thank you a lot. 
![image](https://user-images.githubusercontent.com/52262926/150047681-70aa5bec-9e19-487c-81cb-a14cf8860532.png)

**Comment by Abigale001** [comment]
Hi Jiayin,

Sorry for my late response. All the baselines use the same embedding. Therefore, the comparison with baselines are focusing on the comparison of the proposed framework and the baselines' frameworks. Thank you.
